lib/commons/builder/wikidata_queries.rb

```diff
@@ -17,6 +19,11 @@ def term_condition(term_item_id)
     "?statement pq:P2937 wd:#{term_item_id} ."
   end
 
+  def wd(entity_or_entities)
```

Comment:
This method is here to support a pattern, but I'm not sure it's necessary. Its implementation — checking whether the prefix is already there — is also probably a bit too accepting of sloppy invocation (i.e. not caring whether it's been called on input before calling it again).

Reply:
Ah, that's because it originally had a `return entity_or_entities.map { |e| wd(e) }.join(' ') if entity_or_entities.is_a? Array` line at the top, but I thought that diluted what it was expected to do even more, so removed it (but forgot to rename the parameter :/)